projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"block.sweetmagic.chestnut_stairs": "栗木木板楼梯",
+	"block.sweetmagic.lemon_leaves": "柠檬树叶",
+	"block.sweetmagic.lemon_log": "柠檬原木",
+	"block.sweetmagic.lemon_sapling": "柠檬树苗",
+	"block.sweetmagic.lemon_planks": "柠檬木板",
+	"block.sweetmagic.lemon_slab": "柠檬木板台阶",
+	"block.sweetmagic.lemon_stairs": "柠檬木板楼梯",
+	"block.sweetmagic.orange_leaves": "柑橘树叶",
+	"block.sweetmagic.orange_log": "柑橘原木",
+	"block.sweetmagic.orange_sapling": "柑橘树苗",
+	"block.sweetmagic.orange_planks": "柑橘木板",
+	"block.sweetmagic.orange_planks_mossy": "柑橘木板（生苔）",
+	"block.sweetmagic.orange_planks_huti1": "柑橘木板（刻沿）",
+	"block.sweetmagic.orange_planks_huti3": "柑橘木板（镶边）",
+	"block.sweetmagic.orange_slab": "柑橘木板台阶",
+	"block.sweetmagic.orange_huti_slab": "柑橘木板台阶（刻沿）",
+	"block.sweetmagic.orange_stairs": "柑橘木板楼梯",
+	"block.sweetmagic.orange_huti_stairs": "柑橘木板楼梯（刻沿）",
+	"block.sweetmagic.orange_planks_w": "白色柑橘木板",
+	"block.sweetmagic.orange_planks_w_damage": "白色柑橘木板（磨损）",
+	"block.sweetmagic.orange_planks_w_mossy": "白色柑橘木板（生苔）",
+	"block.sweetmagic.orange_planks_w_huti1": "白色柑橘木板（刻沿）",
+	"block.sweetmagic.orange_planks_w_huti3": "白色柑橘木板（镶边）",
+	"block.sweetmagic.orange_w_slab": "柑橘白色木板台阶",
+	"block.sweetmagic.orange_w_huti_slab": "柑橘白色木板台阶（刻沿）",
+	"block.sweetmagic.orange_w_stairs": "柑橘白色木板楼梯",
+	"block.sweetmagic.orange_w_huti_stairs": "柑橘白色木板楼梯（刻沿）",
+	"block.sweetmagic.coconut_leaves": "椰树树叶",
+	"block.sweetmagic.coconut_log": "椰树原木",
+	"block.sweetmagic.coconut_sapling": "椰树树苗",
+	"block.sweetmagic.coconut_planks": "椰树木板",
+	"block.sweetmagic.coconut_slab": "椰树木板台阶",
+	"block.sweetmagic.coconut_stairs": "椰树木板楼梯",
+	"block.sweetmagic.prism_leaves": "棱镜树叶",
+	"block.sweetmagic.prism_log": "棱镜原木",
+	"block.sweetmagic.prism_sapling": "棱镜树苗",
+	"block.sweetmagic.prism_planks": "棱镜木板",
+	"block.sweetmagic.prism_slab": "棱镜木板台阶",
+	"block.sweetmagic.prism_stairs": "棱镜木板楼梯",
+	"block.sweetmagic.banana_leaves": "香蕉树叶",
+	"block.sweetmagic.banana_sapling": "香蕉树苗",
+	"block.sweetmagic.estor_leaves": "寒带苹果树叶",
+	"block.sweetmagic.estor_log": "寒带苹果原木",
+	"block.sweetmagic.estor_sapling": "寒带苹果树苗",
+	"block.sweetmagic.estor_planks": "寒带苹果木板",
+	"block.sweetmagic.estor_slab": "寒带苹果木板台阶",
+	"block.sweetmagic.estor_stairs": "寒带苹果木板楼梯",
+	"block.sweetmagic.peach_leaves": "桃子树叶",
+	"block.sweetmagic.peach_log": "桃子原木",
+	"block.sweetmagic.peach_sapling": "桃子树苗",
+	"block.sweetmagic.peach_planks": "桃子木板",
+	"block.sweetmagic.peach_slab": "桃子木板台阶",
+	"block.sweetmagic.peach_stairs": "桃子木板楼梯",
+	"block.sweetmagic.magiawood_leaves": "魔法树叶",
+	"block.sweetmagic.magiawood_log": "魔法原木",
+	"block.sweetmagic.magiawood_sapling": "魔法树苗",
+	"block.sweetmagic.magiawood_planks": "魔法木板",
+	"block.sweetmagic.magiawood_slab": "魔法木板台阶",
+	"block.sweetmagic.magiawood_stairs": "魔法木板楼梯",
+	"block.sweetmagic.twilight_alstroemeria": "暮光六出",
+	"block.sweetmagic.sugarbell_plant": "糖铃植株",
+	"block.sweetmagic.sannyflower_plant": "阳光花植株",
+	"block.sweetmagic.moonblossom_plant": "月光植株",
```

```suggestion
	"block.sweetmagic.moonblossom_plant": "月华花植株",
```
这些花的植株/种子到底带不带`花`字？
请全文修改